examples/index.html

```diff
@@ -4,13 +4,23 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Yaga - Leaflet-NG2 | Examples</title>
+    <title>YAGA - leaflet-ng2 | Examples</title>
 
     <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap-theme.min.css" />
 
 </head>
 <body>
+<header id="top" class="navbar navbar-default navbar-static-top navbar-inverse">
+    <div class="container">
+        <ul class="nav navbar-nav">
+            <a class="navbar-brand" href="http://yagajs.org" data-placement="bottom" title="YAGA">YAGA</a>
+            <li class="nav-item nav-link"><a href="http://leaflet-ng2.yagajs.org" title="Home"><span class="fa fa-home" aria-hidden="true"></span>  leaflet-ng2</a></li>
```

Comment:
Sorry, I will correct it again...

Then let's install font-awesome as dev dependency